### PR TITLE
Address  + Postcode - RIP-88

### PR DESCRIPTION
**Pull request by autotelik**
Rationalise duplicated partials for all OrgType's with an address/postcode partial

### CODE REVIEW

**Comment by autotelik** [comment]
:eyes:

File: app/views/flood_risk_engine/shared/_address.html.erb [hunk 1]
```diff
@@ -1,14 +1,14 @@
-<%= t('.postcode_hint') %>
+<%= form.t('.postcode_hint') %>
 <p>
```
Comment (woodpigeon) [inline app/views/flood_risk_engine/shared/_address.html.erb]: Is the `form.t()`  so translations are scoped to the step's yml file and not a generic address.en.yml?

**Comment by woodpigeon** [comment]
Looks good to me but @reggieb might want to review because of potential impact on his address steps eg other?

**Comment by autotelik** [comment]
I  have used  @reggieb version of the partials so should be no regression - only rationalised the location

File: app/views/flood_risk_engine/shared/_address.html.erb (hunk 1 above)
Comment (autotelik) [inline app/views/flood_risk_engine/shared/_address.html.erb]: not entirely sure TBH, this is from the @reggieb version, and as it all worked as is  - without any locale changes to my stuff  - I assume it degrades  ok to generic stuff when no step yml defined

**Comment by woodpigeon** [comment]
@autotelik ok to :ship: then

**Pull request merged**